### PR TITLE
add unit test coverage for apiserver/pkg/admission/plugin/webhook/errors/statuserror_test.go

### PR DESCRIPTION
**Pull request by wongearl**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Improve test coverage

Run test coverage command:

go test -cover ./pkg/admission/plugin/webhook/errors/
Befor:

`ok      k8s.io/apiserver/pkg/admission/plugin/webhook/errors    0.005s  coverage: 86.7% of statements`
After:

`ok      k8s.io/apiserver/pkg/admission/plugin/webhook/errors    0.005s  coverage: 100.0% of statements`
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by leilajal** [comment]
/assign @DangerOnTheRanger 
thanks Kermit!
/triage accepted

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror_test.go [hunk 1]
```diff
@@ -152,3 +155,34 @@ func TestToStatusErr(t *testing.T) {
 		}
 	}
 }
+
+func TestNewDryRunUnsupportedErr(t *testing.T) {
+	type args struct {
+		webhookName string
+	}
+	tests := []struct {
```
Comment (apelisse) [inline staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror_test.go:163]: If you have only one test, you don't specifically need to turn this into a table-test, just compare the thing you want to compare :-)

File: staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror_test.go [hunk 2]
```diff
@@ -152,3 +155,34 @@ func TestToStatusErr(t *testing.T) {
 		}
 	}
 }
+
+func TestNewDryRunUnsupportedErr(t *testing.T) {
+	type args struct {
```
Comment (apelisse) [inline staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/errors/statuserror_test.go:160]: Why is this a struct rather than a plain string?

**Review by apelisse — COMMENTED** [review]
You can make that test much simpler by just testing what you want to test without the table (and the "args" struct).

**Pull request closed without merging**